CHANGELOG.md

```diff
@@ -9,6 +9,7 @@
 
 ## Unreleased
 
+* Allow product name in layout header component when public ([PR #1736](https://github.com/alphagov/govuk_publishing_components/pull/1736))
```

Comment:
We should flag this as a breaking change that there is no longer a default.

Reply:
Added, thanks.